doc/source/docs/user_guide/geocoding.rst

```diff
@@ -54,6 +54,6 @@ a user agent:
 .. attention::
 
     Please consult the Terms of Service for the chosen provider. The example
-    above uses ``'photon'`` (the default), which expects fair usage
+    above uses photon (the default), which expects fair usage
```

Comment:
I'd probably keep this as it was. We refer to the default string being passed to the function.